Makefile

```diff
@@ -63,6 +63,7 @@ start-kvstores:
         etcd -name etcd0 \
         -advertise-client-urls http://0.0.0.0:4001 \
         -listen-client-urls http://0.0.0.0:4001 \
+	-listen-peer-urls http://0.0.0.0:2380 \
```

Comment:
A bit surprising to me, but the other lines here that are part of the continued one-liner have spaces aligning them. Could replace the tab to match that? the alignment is worse in my editor (a tabwidth of 4 makes it worse, I suspect).

Reply:
Done.